phpstan-baseline.neon

```diff
@@ -165,6 +165,16 @@ parameters:
 			count: 1
 			path: Neos.Neos/Classes/Controller/Module/User/UserSettingsController.php
 
+		-
+			message: "#^Parameter \\#2 \\$searchTerm of static method Neos\\\\ContentRepository\\\\Core\\\\Projection\\\\ContentGraph\\\\Filter\\\\SearchTerm\\\\SearchTermMatcher\\:\\:matchesNode\\(\\) expects Neos\\\\ContentRepository\\\\Core\\\\Projection\\\\ContentGraph\\\\Filter\\\\SearchTerm\\\\SearchTerm, Neos\\\\ContentRepository\\\\Core\\\\Projection\\\\ContentGraph\\\\Filter\\\\SearchTerm\\\\SearchTerm\\|null given\\.$#"
+			count: 1
+			path: Neos.Neos/Classes/Controller/Service/NodesController.php
```

Comment:
o.O don't we hide a potential bug by skipping this warning?

Reply:
Thanks for the good qa :D Actually yes how did i miss that 🤯 :O